PopupMenu/PopupMenu.js

```diff
@@ -23,21 +24,27 @@ const PopupMenuBase = kind({
 		onClose: PropTypes.func,
 		onHide: PropTypes.func,
 		open: PropTypes.bool,
+		orientation: PropTypes.oneOf(['horizontal']),
```

Comment:
The idea here is that we technically, at this time, only support one orientation, but want to leave it open to future implementations that allow for vertical scrolling (in the code, and in an updated proptypes), so we only actually list the one that's supported presently. This also directly injects into the className field, so we want to ensure that the prop, if provided is spelled correctly and will work as expected.